pkg/sql/pgwire/testdata/pgtest/param_status

```diff
@@ -128,6 +128,14 @@ ReadyForQuery
 {"Type":"CommandComplete","CommandTag":"SET"}
 {"Type":"ReadyForQuery","TxStatus":"I"}
 
+send
+Query {"String": "DROP ROLE IF EXISTS testuser"}
+----
+
+until ignore_all
```

Comment:
why not use `until ignore=NoticeResponse`?

e.g. https://github.com/cockroachdb/cockroach/blob/cf02adddc78c7f460edb67253f832e97f49b1dd9/pkg/sql/pgwire/testdata/pgtest/copy#L5